src/core/dense_set.h

```diff
@@ -187,6 +187,14 @@ class DenseSet {
       return curr_entry_->HasTtl() ? owner_->ObjExpireTime(curr_entry_->GetObject()) : UINT32_MAX;
     }
 
+    void SetExpiryTime(uint32_t ttl_sec) {
+      if (HasExpiry()) {
+        owner_->ObjUpdateExpireTime(curr_entry_->GetObject(), ttl_sec);
```

Comment:
I was looking if we could squash this with `AddOrUpdate()` but we can't because `AddOrUpdate()` will allocate. But! It's a nice optimization that if the current field exists with expiry we can just update its `ttl_sec` :)